x/bank/app_test.go

```diff
@@ -423,19 +411,6 @@ func TestMsgSetSendEnabled(t *testing.T) {
 			accSeqs:    []uint64{1},
 			expInError: nil,
 		},
-		{
-			desc:       "submitted bad as gov prop",
-			expSimPass: false,
-			expPass:    false,
-			msgs: []sdk.Msg{
-				badGovProp,
-			},
-			accSeqs: []uint64{2},
-			expInError: []string{
-				"invalid denom: bad coin name!",
-				"invalid proposal message",
-			},
-		},
```

Comment:
This case executes in endblocker after proposal pass, cannot catch here.

Reply:
user should be simulating to verify, isnt this enough?

Reply:
just talked with atheesh, We think we should extend simulations to support nested messages because more and more apps are using nested messages with delayed execution

Reply:
This makes sense and would solve the issue indeed